src/styles/utils/_extends.scss

```diff
@@ -1,4 +1,4 @@
 %h1 {
```

Comment:
For now, you don't use it anywhere.
Besides, you have written your font-family in your `h1` selector in _reset.scss file